openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/CollectionDAO.java

```diff
@@ -3255,7 +3255,7 @@ default List<String> listAfter(ListFilter filter, int limit, String afterName, S
       String condition = filter.getCondition();
       if (parentTeam != null) {
         // validate parent team
-        Team team = findEntityByName(parentTeam, filter.getInclude());
+        Team team = findEntityByName(FullyQualifiedName.quoteName(parentTeam), filter.getInclude());
```

Comment:
We need to pass FQN of team not name

Reply:
shouldn't we passing fqn instead of name there?

Reply:
updated with use of fqn